client/src/components/shared/Info/Relations.js

```diff
@@ -23,28 +24,43 @@ type StateProps = {
   toggle: StateUpdater<boolean>,
 }
 
-const Relations = ({data, useNewApi, toggledOn, toggle}: RelationsProps) => {
+const Relations = ({data, name, useNewApi, toggledOn, toggle}: RelationsProps) => {
+  const relationList = orderBy(data, ['edge_types']).map((related: RelatedEntity, i) => {
```

Comment:
Idea: make this into a separate component, I really don't like assigning components to a variable and then using them. But maybe that's just matter of personal preference, so feel free to dismiss this.